utils/github_utils.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+from github import Github
+from flask import current_app as app
+from constants import MASADER_GH_REPO
+
+def report_issue(title: str, message:str):
```

Comment:
Missing space between the argument name and the type-hint `message:str` -> `message: str`.